content/en/os/1.14.x/login/regaining-access/index.markdown

```diff
@@ -83,4 +83,145 @@ The Bottlerocket node should be accessible again.
+    "volumes": [
+        {
+            "name": "apiclient",
+            "host": {
+                "sourcePath": "/usr/bin/apiclient"
+            }
+        },
+        {
+            "name": "apisocket",
+            "host": {
+                "sourcePath": "/run/api.sock"
+            }
+        }
+    ],
+    "requiresCompatibilities": [
+        "EC2"
+    ],
+    "cpu": "1024",
+    "memory": "1024"
+}
+EOF
+```
+
+3. Next, take the file from the previous step and use the AWS CLI to register the task definition. 
+Running the AWS CLI command without any options, will produce a large amount of JSON that is hard to sift through. Instead, use the `query` parameter to get only what you need, format the output text and skip the pager.
+Then take the results and put it into another environment variable. Any errors will be visible as a response in standard output.
+As a confidence check, `echo` the environment variable to make sure it matches what you’d expect from an ARN.
+
+```shell
+export REGAIN_ACCESS_ARN=$(aws ecs register-task-definition \
+  --cli-input-json file://task-def.json \
+  --query "taskDefinition.taskDefinitionArn" \
+  --output text --no-cli-pager)
+```
+
+```shell
+echo $REGAIN_ACCESS_ARN
+```
+
+4. With the task definition ARN stored in the environment variable, you can create the service.
+
+```shell
+aws ecs create-service --cluster "${CLUSTER}" \
+    --task-definition "${REGAIN_ACCESS_ARN}" \
+    --service-name "${SERVICE_NAME}" \
+    --desired-count 1 \
+    --launch-type EC2 \
+    --enable-execute-command \
+    --no-cli-pager
+```
+
+5. For the subsequent step you will need the task ARN which is not included in the response from the `create-service` call.
+The following command will grab all the task ARNs from the service name and cluster you specified earlier (which should only be one), filter the response with the `query` parameter, and use an environment variable to store only what you need.
+Again, do a confidence check to ensure the environment variable looks as you’d expect.
+Wait a few seconds after the previous step and run the following command:
+
+```shell
+export TASK_ARN=$(aws ecs list-tasks --cluster ${CLUSTER} \
+  --service-name ${SERVICE_NAME} \
+  --no-cli-pager \
+  --output text \
+  --query "taskArns[0]")
+```
+
+```shell
+echo $TASK_ARN
+```
+
+6. Finally, you can use the `execute-command` command to re-enable the control or admin container (change `host-containers.control.enabled` to `host-containers.admin.enabled`). The command will output a nearly empty response, but no errors means it worked.
+
+```shell
+aws ecs execute-command --cluster "${CLUSTER}" \
+    --task "${TASK_ARN}" \
+    --container "${CONTAINER_NAME}" \
+    --command "apiclient set host-containers.control.enabled=true" \
+    --interactive
+```
+
+```shell
+Starting session with SessionId: ecs-execute-command-<some hex values>
+
+Exiting session with sessionId:ecs-execute-command-<some hex values>
```

Comment:
As discussed offline, we will want to `exec` into the container and then run `apiclient`, just as in the k8s example.